rest_framework/utils/model_meta.py

```diff
@@ -96,10 +97,18 @@ def _get_forward_relationships(opts):
     """
     forward_relations = OrderedDict()
     for field in [field for field in opts.fields if field.serialize and field.rel]:
+        # For < django 1.6
+        if hasattr(field, 'to_fields'):
+            to_field = field.to_fields[0] if len(field.to_fields) else None
```

Comment:
I'm not that familiar with `to_fields`, why are we only using the first index here?